plugins/modules/ec2_instance.py

```diff
@@ -55,6 +55,22 @@
         Only required when instance is not already present.
     default: t2.micro
     type: str
+  count:
+    description:
+      - Number of instances to launch.
+    default: 1
+    type: int
+  exact_count:
+    description:
+      - An integer value which indicates how many instances that match the 'count_tag' parameter should be running.
+        Instances are either created or terminated based on this value.
+    type: int
+  count_tag:
+    description:
+      - Used with I(exact_count) to determine how many nodes based on a specific tag criteria should be running.
+        This can be expressed in multiple ways and is shown in the EXAMPLES section.  For instance, one can request 25 servers
+        that are tagged with C(class=webserver). The specified tag must already exist or be passed in as the I(instance_tags) option.
+    type: raw
```

Comment:
This module has `filters` which should be able to achieve the same result, I don't think we need a new parameter here.